sql/type_test.go

```diff
@@ -12,9 +12,8 @@ import (
 func TestIsNull(t *testing.T) {
```

Comment:
Seems like this file should be broken up to parallel the breakup of type.go, right?